### PR TITLE
[CARE-6137] Add Coverage Integrations endpoint

### PR DESCRIPTION
**Pull request by marko-ogg**
Merge precondition:
- [x] All the APIs are supported in Prezly

### CODE REVIEW

File: src/types/CoverageIntegration.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import type { CoverageIntegrationRun } from './CoverageIntegrationRun';
+import type { NewsroomRef } from './Newsroom';
+
+type Iso8601DateTime = string;
+
+export enum PROVIDER {
```
Comment (yuriyyakym) [inline src/types/CoverageIntegration.ts:6]: We already have Provider enum in CoverageEntry. It should probably be re-used (with Google Alerts added)

https://github.com/prezly/javascript-sdk/blob/master/src/types/CoverageEntry.ts#L66-L76

**Review by yuriyyakym — APPROVED** [review]

File: src/types/CoverageIntegration.ts (hunk 1 above)
Comment (marko-ogg) [reply src/types/CoverageIntegration.ts]: Good remark 👍🏻  Done in https://github.com/prezly/javascript-sdk/pull/315/commits/397d0b4d86ad6078947ab7900e9becd24b5f366f

File: src/types/CoverageIntegrationRun.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+type Iso8601DateTime = string;
+
+export enum STATUS {
```
Comment (yuriyyakym) [inline src/types/CoverageIntegrationRun.ts:3]:
```suggestion
export enum Status {
```

File: src/types/CoverageIntegrationRun.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    ERROR = 'error',
+    RUNNING = 'running',
+}
+
+export interface CoverageIntegrationRun {
+    id: number;
+    uuid: string;
+    created_at: Iso8601DateTime;
+    updated_at: Iso8601DateTime;
+    status: STATUS;
```
Comment (yuriyyakym) [inline src/types/CoverageIntegrationRun.ts:14]:
```suggestion
    status: Status;
```

File: src/types/CoverageIntegrationRun.ts (hunk 2 above)
Comment (marko-ogg) [reply src/types/CoverageIntegrationRun.ts]: https://github.com/prezly/javascript-sdk/pull/315/commits/24768cadf1b52e9c081c5f4e4d1debc578afcbb5

File: src/types/CoverageIntegrationRun.ts (hunk 3 above)
Comment (marko-ogg) [reply src/types/CoverageIntegrationRun.ts]: https://github.com/prezly/javascript-sdk/pull/315/commits/24768cadf1b52e9c081c5f4e4d1debc578afcbb5

**Pull request merged**